sormas-base/setup/s2s-import-to-truststore.sh

```diff
@@ -52,37 +52,34 @@ else
   fi
 fi
 
-if [[ -z "${SORMAS_PROPERTIES}" ]]; then
-  DEFAULT_SORMAS_PROPERTIES_PATH="${ROOT_PREFIX}/opt/domains/sormas/sormas.properties"
-  if [[ -f "${DEFAULT_SORMAS_PROPERTIES_PATH}" ]]; then
-    SORMAS_PROPERTIES="${DEFAULT_SORMAS_PROPERTIES_PATH}"
+if [[ ! -d "${SORMAS_DOMAIN_PATH}" ]]; then
```

Comment:
sormas domain should not be required for managing the trust store

Reply:
It's changing the `sormas.properties` files when the truststore is created, that's why it is required